satellite/peer.go

```diff
@@ -78,7 +79,8 @@ type Config struct {
 	Identity identity.Config
 
 	// TODO: switch to using server.Config when Identity has been removed from it
-	Server server.Config
+	Server          server.Config
+	SatelliteServer endpoints.Config
```

Comment:
This is confusing when you have flags for satellite like "satellite --server.address xxx --satellite-server.something yyy`